packages/tocco-ui/src/FormattedValue/typeFormatters/HtmlFormatter.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+const HtmlFormatter = props => {
+  const content = props.value
+  if (!content) {
```

Comment:
This just recently changed and gets handled globally:
https://github.com/tocco/tocco-client/blob/master/packages/tocco-ui/src/FormattedValue/FormattedValue.js#L12

(Try to fix this in other components this "sprints-gap")